dev/com.ibm.ws.jsp/resources/com/ibm/ws/jsp/resources/messages.nlsprops

```diff
@@ -1272,3 +1272,15 @@ jsp.error.page.invalid.erroronelnotfound.useraction=
 jsp.isthreadsafe.warning=CWWJS0003W: The Pages 3.1 specification strongly advises Pages developers not to use the deprecated page directive isThreadSafe.
 jsp.isthreadsafe.warning.explanation=SingleThreadModel was removed from Servlet 6.0. Pages 3.1 deprecated isThreadSafe with plans to remove it in a future Pages update.
 jsp.isthreadsafe.warning.useraction=Remove the isThreadSafe directive from all pages.
+
+jsp.javasourcelevel.warning=CWWJS0004W: Requested javaSourceLevel={0} exceeds installed Java {1}, therefore forcing to {1}.
+jsp.javasourcelevel.warning.explanation=File cannot be compiled with a higher source than the available JDK level. 
+jsp.javasourcelevel.warning.useraction=Specify the javaSourceLevel at or below the available JDK level.
+
+jsp.bothsourcelevelset.warning=CWWJS0005W: Specified both javaSourceLevel={0} and jdkSourceLevel={1}. Defaulting to javaSourceLevel={0}
+jsp.bothsourcelevelset.warning.explanation= Only one source level attribute can be used at a time.
+jsp.bothsourcelevelset.warning.useraction= Select one source level attribute. javaSourceLevel is recommended as jdkSourceLevel is stabalized.  
+
+jsp.javasourcelevel.value =CWWJS0006I: The value of the JSP attribute javaSourceLevel is \"{0}\".
+jsp.javasourcelevel.value.explanation =
+jsp.javasourcelevel.value.useraction =
```

Comment:
Can / should we add a explanation / useraction for this one?

Reply:
Since this is just an info message stating the value of `javaSourceLevel`, I don't see any explanation or useraction needed. 

We do not have anything for jdkSourceLevel either: 
```
jsp.jdksourcelevel.value =JSPG8502I: The value of the JSP attribute jdkSourceLevel is \"{0}\".
jsp.jdksourcelevel.value.explanation =
jsp.jdksourcelevel.value.useraction =
```